src/views/Blog.vue

```diff
@@ -57,12 +67,38 @@ export default {
       data.attributes.tags = getTagsArray(get(data, 'attributes.tags', ''));
       await addViewOnBlog(this.blogId);
       this.blog = data;
+
     } catch (error) {
       this.notifyErrors(error);
     }
   },
   methods: {
     get,
+    jsPDF,
+    async getPDF() {
+      this.$refs.pdfContent.innerHTML = this.$refs.blog.innerHTML;
```

Comment:
Is there a need for this assignation?
Can't use just use hte blog innerhtml and remove the hidden diff?